code/modules/projectiles/guns/projectile/pistol/type90.dm

```diff
@@ -0,0 +1,52 @@
+/obj/item/gun/projectile/pistol/type_90
+	name = "OS Type 90 HG .35 Auto \"Mozhu\"" //last emperor
+	desc =	"A Highly advanced gauss pistol produced by onestar right before its collapse. it earned its nickname \"last emperor\" from being the last general issue weapon produced by onestar Uses both standard and highcap .35 Auto mags."
```

Comment:
```suggestion
	desc = "Highly advanced gauss pistol manufactured by the Alliance right before its collapse. Uses both standard and high capacity .35 Auto magazines."
```
OneStar didn't know it would be their last gun when they named it, now did they?

Reply:
they didnt give it the nickname but whatever